src/main/java/org/spongepowered/api/text/format/TextColors.java

```diff
@@ -43,7 +42,7 @@ private TextColors() {
      */
     public static final TextColor NONE = new TextColor() {
 
-        private final Color color = new Color(0, 0, 0, 0);
+        private final Color color = Color.WHITE;
```

Comment:
This is supposed to be simply "no color defined", it's not really related to any color that can be represented in Minecraft directly, it means the color will be inherited.